src/bci_build/package/firefox.py

```diff
@@ -12,11 +12,13 @@
 
 FIREFOX_CONTAINERS = [
     ApplicationStackContainer(
-        name=("kiosk-firefox" if os_version.is_tumbleweed else "kiosk-firefox-esr"),
+        name="kiosk-firefox",
```

Comment:
Why the change after https://github.com/SUSE/BCI-dockerfile-generator/pull/2210#discussion_r1951323930?

Reply:
> is there any plan to provide a non-esr version on sle15?

No, never. I think I added the `-esr` suffix also because on sle16, we'll call the firefox package `firefox-esr` to avoid confusion of customers who were surprised that `zypper in MozillaFirefox` gave them the ESR version and not the "ordinary" one

Reply:
anyhow, I dropped this part of the change, as we have more important things to do.